llvm/test/tools/llvm-objcopy/ELF/set-visibility-check-errors.test

```diff
@@ -1,43 +1,36 @@
-
 # RUN: yaml2obj %s -o %t.o
 # RUN: echo 'foo' > %t.symbols
 
-# Check if passing an invalid visibility type generates an error message
-# RUN: not llvm-objcopy %t.o --set-symbols-visibility=%t.symbols=invalid-type --regex 2>&1 | FileCheck %s --check-prefix=TYPE
-# RUN: not llvm-objcopy %t.o --set-symbol-visibility=foo=invalid-type --regex 2>&1 | FileCheck %s --check-prefix=TYPE
+## Check that passing an invalid visibility type generates an error message
```

Comment:
Nit: still missing full stops ('.' characters) at the end of your test comments.